third_party/half/UPDATING.md

```diff
@@ -0,0 +1,6 @@
+This project is from: https://sourceforge.net/p/half/code/HEAD/tree/
+
+This can be updated by running:
+```shell
+curl http://svn.code.sf.net/p/half/code/trunk/include/half.hpp > half.hpp
```

Comment:
We should be updating the license and README as well, I think

Reply:
Done.